src/FuncSharp/Collections/ReadOnlyList.cs

```diff
@@ -123,11 +123,13 @@ public static IReadOnlyList<T> Create<T>(params T[] values)
         return CreateFlat(values);
     }
 
+    [Obsolete("Use CreateFlat instead. This method will be gone once Mews replaces all the usages.", true)]
```

Comment:
Not sure the `error: true` is the way to go for the deprecation - it essentially forces you to change it on update, not just warns you to do it.